e2e/nomostest/config_sync.go

```diff
@@ -70,6 +70,10 @@ const (
 
 	// clusterRoleName imitates a user-created (Cluster)Role for NS reconcilers.
 	clusterRoleName = "cs-e2e"
+
+	// defaultSyncPollingPeriod is the default override for the git-sync/oci-sync
+	// polling period used by tests. Using a shorter period speeds up the e2e tests.
+	defaultSyncPollingPeriod = 5 * time.Second
```

Comment:
nit: suggest renaming to `shortSyncPollingPeriod` because it is not the real default in production, but default in testing.

Reply:
done